Demo/Tests/AdVideoControls/CaseWithPlayTimeIsLoading.swift

```diff
@@ -20,10 +20,27 @@ class CaseWithPlayTimeIsLoading: SnapshotTest {
         return controller
     }
     
-    func test() {
+   private enum Errors: Error {
```

Comment:
There is a lot of duplication in each test case - can we avoid it somehow?

Reply:
Definitely, I can make `private enum Errors` public